### PR TITLE
Improve readme, add more wiki links

### PR DESCRIPTION
**Pull request by OndraM**
Add more link to wiki, where some part of README were also moved and extended.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Comment by OndraM** [comment]
Review from any native speaker would be greatly appreciated! :slightly_smiling_face: (Maybe @andrewnicols if you'd be able to find some time for this?)

File: README.md [hunk 1]
```diff
@@ -59,6 +57,8 @@ This could be Selenium standalone server, but for local development, you can sen
 
 #### a) Chromedriver
 
+📙 Below you find only simple example, make sure to read our wiki for [more information on Chrome/Chromedriver](https://github.com/php-webdriver/php-webdriver/wiki/Chrome).
```
Comment (andrewnicols) [inline README.md:60]: Minor grammatical suggestion:

Below you _will find a_ simple example. Make sure to read our wiki for more information.

File: README.md [hunk 2]
```diff
@@ -81,34 +81,16 @@ geckodriver
 
 #### c) Selenium standalone server
 
-[Selenium server](https://selenium.dev/downloads/) is useful especially when you need to execute multiple tests at once
-or your tests are run in different browsers - like on your CI server.
-
-Selenium server receives commands and starts new sessions using browser drivers acting like hub distributing the commands
-among multiple nodes.
+Selenium server is useful especially when you need to execute multiple tests at once, if your tests are run
+in different browsers (like on your CI server) or eg. when you need to distribute tests amongst multiple machines
```
Comment (andrewnicols) [inline README.md:85]: Selenium server can be useful when you need to execute multiple tests at once, when you run tests in several different browsers (like on your CI server), or when you need to distribute tests amongst several machines in grid mode (where one Selenium server acts as a hub, and others connect to it as nodes).

File: README.md [hunk 3]
```diff
@@ -81,34 +81,16 @@ geckodriver
...
+in grid mode (one Selenium server hub and multiple Selenium nodes).
 
-To run the standalone server, download [`selenium-server-standalone-#.jar` file](http://selenium-release.storage.googleapis.com/index.html)
-(replace # with the current server version). Keep in mind **you must have Java 8+ installed**.
+Selenium server then act like a proxy and takes care of distributing commands to the respective nodes.
 
-Run the server:
+The latest version can be found on [Selenium download page](https://www.selenium.dev/downloads/).
```
Comment (andrewnicols) [inline README.md:90]: ... on _the_ Selenium download page.

File: README.md [hunk 4]
```diff
@@ -161,14 +143,39 @@ $desiredCapabilities->setCapability('acceptSslCerts', false);
 // Run headless firefox
 $desiredCapabilities->setCapability('moz:firefoxOptions', ['args' => ['-headless']]);
 
-$driver = RemoteWebDriver::create($host, $desiredCapabilities);
+$driver = RemoteWebDriver::create($serverUrl, $desiredCapabilities);
 ```
 
-They can also be used to [configure proxy server](https://github.com/php-webdriver/php-webdriver/wiki/HowTo-Work-with-proxy) which the browser should use.
-To configure Chrome, you may use ChromeOptions - see [details in our wiki](https://github.com/php-webdriver/php-webdriver/wiki/ChromeOptions).
+Capabilities can also be used to [📙 configure proxy server](https://github.com/php-webdriver/php-webdriver/wiki/HowTo-Work-with-proxy) which the browser should use.
```
Comment (andrewnicols) [inline README.md:149]: configure _a_ proxy server

**Review by andrewnicols — CHANGES_REQUESTED** [review]
Looks like a good set of updates. Just a few minor grammatical uggestions.

**Review by williamdes — APPROVED** [review]

**Comment by OndraM** [comment]
Thanks a lot @andrewnicols 👌.

**Pull request merged**